example/v1alpha1/main.go

```diff
@@ -51,9 +52,10 @@ func main() {
 
 	_, oID, pErr := name.ParseOccurrence(createdO.Name)
 	if pErr != nil {
-		log.Fatalf("Unable to get occurenceId from occurrence name %v: %v", createdO.Name, err)
+		log.Fatalf("Unable to get occurenceId from occurrence name %v: %v", createdO.Name, pErr)
 	}
-	if got, _, err := client.GetOccurrence(oPID, oID); err != nil {
+
+	if got, _, err := client.GetOccurrence(nPID, oID); err != nil {
```

Comment:
This should be nPID. The Occurrence exists in the Occurrences project.

Reply:
This is oPID in the last commit